spec/dummy/app/themes/default/views/thredded/shared/_top_nav.html.erb

```diff
@@ -2,31 +2,39 @@
   <ul>
     <!-- always -->
     <li class="user-navigation--search">
-      <!-- <svg viewBox="0 0 32 32"><use xlink:href="#search1"></use></svg> -->
```

Comment:
Cool cool man I think that'd be great. The icons I had there can probably get the axe and I agree - they're NOT great!

Once you're ready to place new ones in here, do your thing. I just took these out because these sorts of comments are just noise, in my opinion.

Which brings me to ...